protocol-definitions/MC.yaml

```diff
@@ -222,3 +222,96 @@ methods:
             8 - FREE_NATIVE_MEMORY_SIZE
             9 - FREE_NATIVE_MEMORY_PERCENTAGE
     response: {}
+  - id: 5
+    name: getMemberConfig
+    since: 2.0
+    doc: |
+      Gets the effective config of a member rendered as XML.
+    request:
+      retryable: true
+      acquiresResource: false
+      partitionIdentifier: -1
+      params: []
+    response:
+      params:
+        - name: configXml
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Effective config of a member rendered as XML.
+  - id: 6
+    name: runGc
+    since: 2.0
+    doc: |
+      Requests garbage collection to be performed on the member's JVM.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: -1
+      params: []
+    response: {}
+  - id: 7
+    name: getThreadDump
+    since: 2.0
+    doc: |
+      Takes a thread dump of the member's JVM.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: -1
+      params:
+        - name: dumpDeadLocks
+          type: boolean
+          nullable: false
+          since: 2.0
+          doc: |
+            Whether only dead-locked threads or all threads should be dumped.
+    response:
+      params:
+        - name: threadDump
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Thread dump of the member's JVM.
+  - id: 8
+    name: shutdownMember
```

Comment:
How does this method decide which member to shutdown?
From a protocol perspective, it's not clear. A unisocket or smart client may have different behaviour.
A member uuid parameter or a better documentation can be added.

Reply:
`Shuts down the member it's called on.` -> is this clear enough as documentation? Will add the same to the other which have the same behavior.

Reply:
Yes the definition is clear for me.
In order to shut down a member,  you'll need to open a connection to it. That drops any unisocket support. 
If that's the case, we may put a note at the top as this service only supports smart client.

Reply:
We only use smart client. Will add the note to the service about unisocket client.

Reply:
Thinking again, I don't see a reason why we should drop support for unisocket clients. We might need them for whatever reason in the future. I will add the `member` parameter to the operations that need it (also to the ones I added in my previous PR).

Reply:
After a discussion with Asim, we have decided to only document the requirement of smart client and keep the protocol definitions as they are.

Reply:
I've updated the documentation.